prototests/test_splineBasis.py

```diff
@@ -0,0 +1,73 @@
+# content of ./test_parametrize.py
+import numpy as np
+from neurostatslib.basis import BSplineBasis, Cyclic_BSplineBasis
+
+def pytest_generate_tests(metafunc):
+    # called once per each test function
+    funcarglist = metafunc.cls.params[metafunc.function.__name__]
+    argnames = sorted(funcarglist[0])
+    metafunc.parametrize(
+        argnames, [[funcargs[name] for name in argnames] for funcargs in funcarglist]
+    )
+
+
+class TestBspline:
```

Comment:
there currently is no Bspline, right? so this doesn't belong here -- why doesn't it make the tests fail?

Reply:
I think this folder is left over from earlier, should be deleted (copy contents to an issue or keep a copy locally if it will be useful later)

Reply:
removed